htlcswitch/link_test.go

```diff
@@ -3913,3 +3913,377 @@ func restartLink(aliceChannel *lnwallet.LightningChannel, aliceSwitch *Switch,
 
 	return aliceLink, t, cleanUp, nil
 }
+
+// TestChannelLinkNoMoreUpdates tests that we won't send a new commitment
+// when there's no new updates to sign.
+func TestChannelLinkNoMoreUpdates(t *testing.T) {
```

Comment:
This is a great test! However, I think we can put in a bit of upfront work here to add some helper functions to make the test a bit more readable and reduce a bit of vertical space here.